src/common/Main/styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const MainContainer = styled.main`
+  max-width: 1368px;
+  min-height: 1391px;
```

Comment:
@mateusz24polok Czy ten min-height jest tutaj potrzebny? I to taki na sztywno? Nie powinien zależeć po prostu od zawartości? 
Jak to się ma do różnych rozdzielczości ekranów i np telefonów?

Reply:
ooo faktycznie, nie zauważyłem tego wcześniej.
Zupełnie niepotrzebne. 

Taka w takim razie uwaga, jeśli chodzi o height to bardzo ale to bardzo rzadko jest konieczność jego stosowania.

Reply:
Oczywiście, macie rację, już poprawione ;)